src/App.tsx

```diff
@@ -1,7 +1,52 @@
-import { greet } from "./utils/greet";
+//import EpisodesList from "./components/EpisodesList";
+import episodes from "./episodes.json";
+import AppHeader from "./components/AppHeader";
+import StructureEpisode from "./components/StructureEpisode";
+import { useState } from "react";
+import AppFooter from "./components/AppFooter";
+
+export interface IShorterEpisode {
+  id: number;
+  name: string;
+  season: number;
+  number: number;
+  airdate: string;
+  airtime: string;
+  image: {
+    original: string;
+    medium: string;
+  };
+  summary: string;
+}
```

Comment:
Perhaps we should move this type out to types so it can be shared among the other components.